### PR TITLE
Please add coq-functional-algebra to the Coq Package Index

### PR DESCRIPTION
**Pull request by llee454**
Hi,

Please consider adding the Functional-Algebra library (https://github.com/llee454/functional-algebra) to the Coq Package Index. This library contains over 800 definitions and proofs for abstract algebra, but more importantly, it presents an example of functional programming in Coq and includes certified functions for simplifying algebraic expressions and proving algebraic equations. I believe that these proofs, functions, and examples are valuable and will benefit the larger Coq community.

Please let me know if there are any improvements that I can make,

- Larry Lee <llee454@gmail.com>

### CODE REVIEW

File: released/packages/coq-functional-algebra/coq-functional-algebra.1.0.0/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/llee454/functional-algebra/archive/1.0.2.tar.gz"
```
Comment (gares) [inline released/packages/coq-functional-algebra/coq-functional-algebra.1.0.0/url]: The archive is named 1.0.2, but the package version is 1.0.0

**Comment by llee454** [comment]
Hi Gares,

Thanks for reviewing my pull request. I've renamed the package to reflect the latest release it's libraries latest release. I'll update my request momentarily.

Thanks,
- Larry

**Comment by llee454** [comment]
OK, I've updated the package version number. Please let me know if I need to do anything else.

**Comment by llee454** [comment]
Thanks Gares!

**Pull request merged**